doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,114 +3,103 @@
+You can now submit a change request from one branch to another branch of the same project.
 
-Three reference implementations are available that you can use as starter applications. They are provided in the *{PRODUCT} {PRODUCT_VERSION_LONG} Reference Implementations* download, available on the {PRODUCT} Software Downloads page in the Red Hat Customer Portal.
+=== Test Scenarios
 
-.Employee Rostering reference implementation
-The employee rostering reference implementation enables you to create an application that assigns employees to shifts on various positions in an organization. For example, you can use the application to distribute shifts in a hospital between nurses, guard duty shifts across a number of locations, or shifts on an assembly line between workers.
+The test scenarios designer in {CENTRAL} is now available with the new features supported in {PRODUCT} {PRODUCT_VERSION}.
 
-.High available event-driven decisioning reference implementation
-A reference implementation for high available event-driven decisioning on {OPENSHIFT} is available. You can customize this reference implementation to deploy {PRODUCT_DROOLS} engine code that requires stateful processing, including rules developed with complex event processing, in an OpenShift environment. Doing this enables the {DECISION_ENGINE} to process complex event series with high availability.
+* If the same data is shared by multiple test scenarios, you can use the *Background* tab to define them once for rules-based as well as DMN-based test scenarios. You can add and define the `GIVEN` data which is common for the dedicated test scenario simulation, based on the available data objects.
 
-.Vehicle route planning reference implementation
-The vehicle route planning reference implementation enables you to create an application that solves a vehicle route planning problem with real-world maps, roads, and vehicles delivering goods to locations, each with a specific capacity. For more information, see the README file in the vehicle routing ZIP file, included in the reference implementation download.
+* In rule-based test scenarios, you can use the MVEL expression with the prefix *#* to call a java method to a property or instance header cell during test scenarios execution.  
 
-== {CENTRAL}
+=== Generic error dialog improvement in {CENTRAL}
+
+A generic error dialog is a pop-up window that notifies users about different runtime errors that are produced in {CENTRAL}.
+The following list provides a summary of generic error dialog enhancements:
+
+* When an error is generated, the error is logged in the server logs. You can fetch and retrieve the generated errors if required.
+* You can see the stack trace of the generated error in the popup itself.
+* To suppress the error notifications temporarily, you can select the *Don't show this message again for* checkbox and select the time duration from the dropdown. However, the error messages are still logged in the server logs.
```

Comment:
delete 'you can':
To suppress the error notifications temporarily,  select.

dropdown is not a noun. ...drop-down list.

IBM SG:
drop-down list:
A list of selectable items that is displayed when the user clicks an arrow button.